evm/src/cpu/kernel/interpreter.rs

```diff
@@ -238,6 +238,9 @@ impl<'a> Interpreter<'a> {
             0x09 => self.run_mulmod(),                                  // "MULMOD",
             0x0a => self.run_exp(),                                     // "EXP",
             0x0b => todo!(),                                            // "SIGNEXTEND",
+            0x0c => todo!(),                                            // "ADDFP254",
+            0x0d => todo!(),                                            // "MULFP254",
+            0x0e => todo!(),                                            // "SUBFP254",
```

Comment:
wouldn't these just amount to the implementations for `run_{add,sub,mul}mod` but with `N` hardcoded to the p254?

Reply:
You’re right, but I’d just rather do it in a different PR to keep this one small. (Also, I’ll need to write `run_submod` because it doesn’t currently exist.)